app/controllers/invoices_controller.rb

```diff
@@ -27,9 +27,12 @@ def update
 
   def check
     authorize(Invoice)
-    @company = Company.find_by!(code: params[:code])
 
-    return head :unprocessable_entity unless policy_scope(Invoice).all?(&:paid?)
+    if policy_scope(Invoice).all?(&:paid?)
+      @company = Company.find_by!(code: params[:code])
+    else
+      return head :unprocessable_entity
+    end
```

Comment:
```suggestion
    return head :unprocessable_entity if policy_scope(Invoice).all?(&:paid?) 
    
    @company = Company.find_by!(code: params[:code])
```

Reply:
Hey @m-pereira I make the suggested changes and also double check if all the tests were covered, below you can see a screen shot of it.

![image](https://user-images.githubusercontent.com/92229784/156755005-218689eb-1cc9-4a3d-9290-74193c2bf2f4.png)

Reply:
Actually, we're using [simplecov](https://github.com/simplecov-ruby/simplecov) to ensure the coverage. Can you please check if there is a test for this action

Reply:
There are tests for this action (Check) and simplecov shows that 98.21% are covered.